### PR TITLE
Refactor Awesome Books with ES6 features

### PR DESCRIPTION
**Pull request by yash244466666**
Added the following files for the app

- [books-bg.png](https://github.com/yash244466666/Awesome-books-with-ES6/blob/addCodesAndElemnts/res/books-bg.png)
- [Book-storage.js](https://github.com/yash244466666/Awesome-books-with-ES6/blob/addCodesAndElemnts/modules/Book-storage.js)
- [luxon.js](https://github.com/yash244466666/Awesome-books-with-ES6/blob/addCodesAndElemnts/modules/luxon.js)
- [nav-menu.js](https://github.com/yash244466666/Awesome-books-with-ES6/blob/addCodesAndElemnts/modules/nav-menu.js)
- [store.js](https://github.com/yash244466666/Awesome-books-with-ES6/blob/addCodesAndElemnts/modules/store.js)
- [index.css](https://github.com/yash244466666/Awesome-books-with-ES6/blob/addCodesAndElemnts/index.css)
- [index.html](https://github.com/yash244466666/Awesome-books-with-ES6/blob/addCodesAndElemnts/index.html)
- [index.js](https://github.com/yash244466666/Awesome-books-with-ES6/blob/addCodesAndElemnts/index.js)
- [README.md](https://github.com/yash244466666/Awesome-books-with-ES6/blob/addCodesAndElemnts/README.md)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    if (el.classList.contains('delete')) {
+      el.parentElement.parentElement.remove();
+    }
+  }
+
+  static clearField() {
+    document.querySelector('#title').value = '';
+    document.querySelector('#author').value = '';
+  }
+}
```
Comment (Qoosim) [inline index.js]: - [ ] Kindly adopt the usage of arrow function for all the functions in your project as this is part of project requirements. Please, take a look :point_down: 

### Project requirements
arrow-funct

File: modules/store.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  static removeBook(authore) {
+    const books = store.getBooks();
+    books.forEach((book, index) => {
+      if (book.authore === authore) {
+        books.splice(index, 1);
+      }
+    });
+
+    localStorage.setItem('books', JSON.stringify(books));
+  }
```
Comment (Qoosim) [inline modules/store.js]: - [ ] Kindly adopt the usage of arrow function for all the functions in your project as this is part of project requirements. Please, take a look :point_down: 

### Project requirements
arrow-funct

**Review by Qoosim — CHANGES_REQUESTED** [review]
#    *Status: Required Changes* :recycle: 

Hi @yash244466666, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Correct Git Flow. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Kindly make your PR title descriptive of what you worked on the PR. At this moment, your pull request title does not reflect what you worked on the PR. Kindly change it to something like. `Refactor Awesome Books with ES6 features` or anything related. 

### Your PR title
your-pr-title


- [OPTIONAL] I made this optional for you not to have some complicated issues if not handle properly. Your feature-branch name must also reflect the type of work you do on that branch. For the future purpose, always try to make your branch name descriptive too. These are best practices to embrace at the start of ones journey as a software developer. I wish you the best brother. :handshake:

**Comment by yash244466666** [comment]
Add changes as instructed

- [x] Made changes in [index.js](https://github.com/yash244466666/Awesome-books-with-ES6/blob/addCodesAndElemnts/index.js) as instructed.
- [x] Made changes in [store.js](https://github.com/yash244466666/Awesome-books-with-ES6/blob/addCodesAndElemnts/modules/store.js) as instructed.
- [x] Made changes in PR title as instructed.

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @yash244466666 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - No linter errors. ✔️
 - Correct Git Flow. ✔️
 - Good README file. ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**